lib/Backends/CMakeLists.txt

```diff
@@ -1,20 +1,16 @@
-add_subdirectory(Interpreter)
-
-LIST(APPEND linked_device_managers InterpreterDeviceManager)
-LIST(APPEND linked_factories InterpreterFactory)
-
-if(GLOW_WITH_OPENCL)
-  add_subdirectory(OpenCL)
-  LIST(APPEND linked_factories OpenCLFactory)
-  LIST(APPEND linked_device_managers OpenCLDeviceManager)
-endif()
-
-
-if(GLOW_WITH_CPU)
-  add_subdirectory(CPU)
-  LIST(APPEND linked_factories CPUFactory)
-  LIST(APPEND linked_device_managers CPUDeviceManager)
-endif()
+# Iterate all subdirectories and check whether
+# particular backend is enabled.
```

Comment:
Strange formatting. Why 2 lines instead of one?

Reply:
no particular reason. can be made as a single line.